package.json

```diff
@@ -52,6 +52,7 @@
     "*.{js,jsx,ts,tsx,md,scss}": "prettier --write"
   },
   "dependencies": {
-    "@motion-canvas/core": "^3.0.0"
+    "@motion-canvas/core": "^3.0.0",
+    "glob": "^9.3.2"
```

Comment:
`glob` should not be a top-level dependency.
Packages are published individually with their respective `package.json` included.
You should run:
```
npm i glob -w packages/vite-plugin
```
To install `glob` as the dependency of the `create` package instead.

Reply:
done!